engine/engine/content/wscript

```diff
@@ -40,6 +40,7 @@ def build(bld):
     builtins = bld(features='barchive',
                    source_root='default/content',
                    content_root='../content',
+                   project_root='content',
```

Comment:
Having two variables pointing out the same path seems a bit odd.

What was the issue with using "content_root"? That it starts with "../" ?